modules/services/localization/src/main/res/values/strings.xml

```diff
@@ -1389,6 +1389,7 @@
     <string name="end_of_year_prompt_card_title" translatable="false">@string/end_of_year_launch_modal_title</string>
     <string name="end_of_year_prompt_card_summary">See your top podcasts, categories, listening stats and more.</string>
     <string name="end_of_year_share_via" translatable="false">@string/podcasts_share_via</string>
+    <string name="end_of_year_listening_time">In 2022, you spent %1$s listening to podcasts.</string>
```

Comment:
Nitpick: I don't think we're actually telling them how much time they spent listening to podcasts because we're not taking account of skipping forward/backward or playback speed. I think we're more telling them that they listened to XXX hours/minutes of podcasts.

For example, I can make the total time go down if I rewind a podcast or make a previously played podcast as unplayed. 

Just raising this as something worth keeping in mind. Definitely not a blocker.

Reply:
I understand your point, @mchowning. AFAIU, it is just an approximate listening time, does not take into account changes to played time due to skipping, and matches iOS behavior. 

I'll keep it in mind for sure. 👍